README.md

```diff
@@ -16,16 +23,178 @@ There have been, and are currently, several efforts in this area:
 
 This repository is an effort to bring together tool owners to standardize on a common specification for a description format.
 
-# Use Cases
+Please join the discussion in [Issues](https://github.com/webcomponents/custom-elements-json/issues).
 
-## Editor Support
+## Versioning
 
-Developers using custom elements should be able to get full-featured IDE support including auto-competion, hover-documentation, unknown symbol warnings, etc. These features should be available in HTML files, and in various template syntaxes via template-specific tools.
+The `custom-elements.json` follows a version number to allow for a gradual upgrade process.
 
-## Documentation
+- [Version 2](#version-2)
+- [Version 1](#version-1)
+
+### Version 2
+
+The main goal if this version is to address all key points of web-components.
+It shall serve as a starting point for adoption in tools and for web component authors.
+
+It is fully based on [Version 1](#version-1). 
+It adds all mentioned key points for web components.
+It on purpose does not define a type system. This is left to the next iteration.
+Therefore "type" should be seen as a type hint which can be presented to the end-user as documentation.
+
+Following is an example which shows all available capabilities.
+
+```json
+{
+  "version": 2,
+  "tags": [
+    {
+      "name": "demo-wc-card",
+      "description": "This is a container looking like a card with a back and front side you can switch",
+      "attributes": [
+        {
+          "name": "header",
+          "type": "string",
+          "description": "Shown at the top of the card",
+          "default": "Your Message",
+          "required": true
+        }
+      ],
+      "properties": [
+        {
+          "name": "header",
+          "type": "string",
+          "description": "Shown at the top of the card",
+          "default": "Your Message",
+          "required": true
+        }
+      ],
+      "events": [
+        {
+          "name": "side-changed",
+          "description": "Fires whenever it switches between front/back"
+        }
+      ],
+      "methods": [
+        {
+          "name": "toggle",
+          "description": "Switches between front and back card"
+        }
+      ],      
+      "slots": [
+        {
+          "name": "",
+          "description": "Content inside the card which displayed on the front page"
+        }
+      ],
+      "cssProperties": [
+        {
+          "name": "--demo-wc-card-front-color",
+          "description": "Font color for the front",
+          "type": "color"
+        }
+      ],
+      "cssShadowParts": [
```

Comment:
Since "CSS Custom Properties" was shortened (presumably for brevity), would this be better as "CSS Parts"?

Reply:
I would rather go for renaming `cssProperties` to `cssCustomProperties`? better be explicit? what do you think?

Reply:
Yah, I would go one way or the other. Concise or pedantic, not a mix of both :)

Reply:
thx adjusted to `cssCustomProperties` 👍